content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -180,7 +180,23 @@ for database debugging.
     PONG
     ```
 
+### Optionaly let _kubectl_ choose the local port
```

Comment:
There's a spelling error, and I would also override the automatic choice of fragment identifier:
```suggestion
### Optionally let _kubectl_ choose the local port {#let-kubectl-choose-local-port}
```